internal/component/otelcol/exporter/debug/debug_test.go

```diff
@@ -0,0 +1,83 @@
+package debug_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/grafana/agent/internal/component/otelcol/exporter/debug"
+	"github.com/grafana/river"
+	"github.com/stretchr/testify/require"
+	otelcomponent "go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configtelemetry"
+	debugexporter "go.opentelemetry.io/collector/exporter/debugexporter"
+)
+
+func Test(t *testing.T) {
+	tests := []struct {
+		testName       string
+		args           string
+		expectedReturn debugexporter.Config
+		errorMsg       string
+	}{
+		{
+			testName: "defaultConfig",
+			args: `
+			verbosity = "basic"
+			sampling_initial = 2
+			sampling_thereafter = 500
```

Comment:
```suggestion
```

In order to truly test if the defaults are set correctly, we need to not supply any arguments.